core/exec/src/mill/exec/Execution.scala

```diff
@@ -146,9 +147,12 @@ private[mill] case class Execution(
               )
 
               val verboseKeySuffix = s"/${terminals0.size}"
-              logger.setPromptHeaderPrefix(s"$countMsg$verboseKeySuffix")
-              if (failed.get()) None
-              else {
+              logger.setPromptHeaderPrefix(s"$countMsg$verboseKeySuffix${
+                  mill.internal.Util.formatFailedCount(rootFailedCount.get())
+                }")
```

Comment:
Let's move this big string into a local `def` that we use in these two places to remove the duplication